apps/backoffice-v2/src/domains/auth/hooks/mutations/useSignOutMutation/useSignOutMutation.tsx

```diff
@@ -18,9 +18,7 @@ export const useSignOutMutation = () => {
       void queryClient.cancelQueries();
     },
     onSuccess: (data, { callbackUrl, redirect }) => {
-      queryClient.setQueryData(authenticatedUser.queryKey, {
-        user: undefined,
-      });
+      void queryClient.invalidateQueries();
```

Comment:
`queryClient.invalidateQueries` is already done in `onSettled` so it can be removed in `onSuccess`. Please re-add the `setQueryData` as `invalidateQueries` may take a long time to finish. 🙏